service/policy/db/migrations/20241125220354_keys_table.sql

```diff
@@ -0,0 +1,233 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS
+    public_keys (
+        id uuid DEFAULT gen_random_uuid () PRIMARY KEY,
+        is_active boolean NOT NULL DEFAULT TRUE,
+        was_used boolean NOT NULL DEFAULT FALSE,
+        key_access_server_id uuid NOT NULL REFERENCES key_access_servers (id),
+        key_id varchar(36) NOT NULL,
+        alg varchar(50) NOT NULL,
+        public_key text NOT NULL,
+        metadata jsonb,
+        created_at timestamp,
+        updated_at timestamp,
+        UNIQUE (key_access_server_id, key_id, alg),
```

Comment:
Is the intended behavior here that we want to allow a single KAS to have multiple keys of different algorithms with the same ID? 

```
KAS_A + kid_123 + rsa:2048
KAS_A + kid_123 + secp:256
```

Reply:
If so, or if the desired intention is different and I am misunderstanding, can we please comment in the migration that this is the case?